Readme.md

```diff
@@ -133,6 +133,20 @@ Note: see https://github.com/thomseddon/node-oauth2-server/tree/master/examples/
  - *mixed* **error**
      - Truthy to indicate an error
 
+#### checkScope (accessToken, requiredScope, callback)
```

Comment:
I made the name explicit here to avoid confusion between the scope already associated with the token, and the scope required to access the route. I suppose the comment will suffice to clarify that though; I'll probably change it.